### PR TITLE
Update Senior Engineer JD

### PR DESCRIPTION
**Pull request by norfy78**
I have updated some of the text to reflect changes in the last 6 months and the tech stack.

### CODE REVIEW

File: roles/senior_engineer.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Our Senior Engineers are crazy about software, have expertise in solving a wide
 
 ## What does the job entail?
 
-We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure on AWS with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a Continuous Delivery pipeline running our automated test suites before deploying changes.
+We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript and C# these days. We will usually be responsible for setting up a customer’s infrastructure on AWS, GCP or Azure with tools like Ansible and Terraform though sometimes we opt for a Platform as a Service like Heroku. We make sure there is a CI/CD pipeline running our automated test suites before deploying changes.
```
Comment (chrisblackburn) [inline roles/senior_engineer.md]: I think this makes it sound like we're C# backend and React/JS frontend nowadays, which isn't quite right, or the message I think we should convey to potential hires. I actually never liked this sentence at all. I'd suggest something such as:

`We build solutions in a variety of languages and platforms, including Ruby on Rails, React, JavaScript and C# - whatever's the right tool for the job.`

Comment (yaseminercan) [reply roles/senior_engineer.md]: Yes I like that. makes it more timeless

File: roles/senior_engineer.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ Further required experience:
 - Pair programming as we pair around 50% of the time
 - Agile practices such as Scrum, XP, and/or Kanban
 - Working with a range of SQL and NoSQL databases
-- Infrastructure as code tools such as Ansible, Chef, Puppet, and/or Saltstack
+- Infrastructure as code tools such as Ansible, Chef, Puppet, Terraform and/or Saltstack
 - Debugging a range of applications
-- Experience debugging infrastructure (AWS/Heroku/Linux/HTTP)
+- Experience debugging infrastructure (AWS/Heroku/Linux/HTTP, GCP and Azure)
```
Comment (chrisblackburn) [inline roles/senior_engineer.md]: Should probably be consistent formatting:

AWS/Heroku/Linux/HTTP/GCP/Azure or
AWS, Heroku, Linux, HTTP, GCP, and Azure

File: roles/senior_engineer.md [hunk 3]
```diff
@@ -36,19 +36,19 @@ Everything else is optional but highly sought after. We would love you to have e
 
 We are a sanctuary for those wanting to hone their skills alongside like-minded learners. After joining our team it is common for new starters to comment on how much they’re learning and how much they enjoy the fact they are surrounded by people they can learn from. This includes our most senior hires.
 
-The biggest thing you’ll take away from our culture after spending the day with us would be that we practice continuous improvement at every level. Everyone has peer-based one-to-ones every 2 weeks, along with monthly one-to-ones with a director. Teams have fortnightly retrospectives. We also hold a company wide retrospective fortnightly. We discuss our problems out in the open, and rather than punish failure we band together to find solutions.
+The biggest thing you’ll take away from our culture after spending the day with us would be that we practice continuous improvement at every level. Everyone has a one-to-one each month with a mentor, twice a month during probation. We discuss our problems out in the open, and rather than punish failure we band together to find solutions.
```
Comment (chrisblackburn) [inline roles/senior_engineer.md]: I could be wrong, but I think most people in the business are having line management sessions every two weeks?

**Pull request closed without merging**